pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -103,11 +94,17 @@ in stdenv.mkDerivation rec {
     done
   '';
 
+# GSETTINGS_SCHEMAS_PATH includes gtk twice, so wrapProgram with ${wxGTK.gtk} is used instead of wrapGApp
+# since wrapGApp needs most of these set explicitly anyway (with dontWrapGApps = true;), lets not depend on it
   preFixup = ''
     buildPythonPath "$out $pythonPath"
     gappsWrapperArgs+=(--set PYTHONPATH "$program_PYTHONPATH")
```

Comment:
doh, this line isn't used anymore, the --set should either be added to wrapProgram
or it's not needed, you found scripting to work?

Reply:
I haven't given it thorough testing, but yes it appears to work.

Reply:
i can't launch the python shell from pcbnew, got something working, just cleaning up the commit

Reply:
ok, fixed [here](https://github.com/Evils-Devils/nixpkgs/commit/500a1bf8715073263535101442d6109ca758aea7)
also had to wrap pcbnew, otherwise python doesn't work if you run it directly

Reply:
looks good to me. i've included it in the pr